stable/prometheus-operator/README.md

```diff
@@ -330,7 +330,8 @@ For a full list of configurable values please refer to the [Grafana chart](https
 | `grafana.sidecar.dashboards.enabled` | Enable the Grafana sidecar to automatically load dashboards with a label `{{ grafana.sidecar.dashboards.label }}=1` | `true` |
 | `grafana.sidecar.dashboards.label` | If the sidecar is enabled, configmaps with this label will be loaded into Grafana as dashboards | `grafana_dashboard` |
 | `grafana.sidecar.datasources.enabled` | Enable the Grafana sidecar to automatically load dashboards with a label `{{ grafana.sidecar.datasources.label }}=1` | `true` |
-| `grafana.sidecar.datasources.defaultDatasourceEnabled` | Enable Grafana `Prometheus` default datasource` | `true` |
+| `grafana.sidecar.datasources.defaultDatasourceEnabled` | Enable Grafana `Prometheus` default datasource | `true` |
+| `grafana.sidecar.datasources.statefulSetsAsDatasources` | Create datasoruce for each Pod of Prometheus StatefulSet i.e. `Prometheus-0`, `Prometheus-1` | `false` |
```

Comment:
Since there is only a single stateful set that will be added as datasources, perhaps this should be called `statefulSetAsDatasources` or `createPrometheusReplicasDatasources`

Reply:
Oops! you are right. Going with `createPrometheusReplicasDatasources` as it makes things more clear.